framework_tvb/tvb/interfaces/web/controllers/spatial/local_connectivity_controller.py

```diff
@@ -44,6 +44,7 @@
 from tvb.core.entities.storage import dao
 from tvb.core.neocom import h5
 from tvb.core.adapters.abcadapter import ABCAdapter
+from tvb.core.services.exceptions import ServicesBaseException
```

Comment:
This is a service layer exception and we are in a controller.
For sure we have exceptions for controllers. If none appropriate exist, just create one.